Casks/inkstitch.rb

```diff
@@ -30,7 +30,9 @@
   end
 
   preflight do
-    system_command "/bin/mkdir", args: ["-p", "#{ENV["HOME"]}/Library/Application Support/org.inkscape.Inkscape/config/inkscape/extensions"]
+    system_command "/bin/mkdir",
+                   args: ["-p", "#{ENV["HOME"]}/Library/Application Support \
+                   /org.inkscape.Inkscape/config/inkscape/extensions"]
```

Comment:
```suggestion
                   args: [
                     "-p",
                     "#{ENV["HOME"]}/Library/Application Support/org.inkscape.Inkscape/config/inkscape/extensions",
                   ]
```
Seems easier to read. Unless this isn't standard in cask.

Reply:
Ah, wait, this probably still violates the line length limit.